src/common/snippets/src/lowered/pass/propagate_subtensors.cpp

```diff
@@ -136,8 +136,8 @@ void propagate_updated_subtensor_through_loop(const LinearIR& linear_ir,
             expr_it = inner_end;
             continue;
         }
-        if ((ov::is_type<snippets::op::BroadcastMove>(expr_it->get()->get_node()) ||
-            ov::is_type<snippets::op::BroadcastLoad>(expr_it->get()->get_node())) &&
+        if ((ov::is_type_any_of<snippets::op::BroadcastMove, snippets::op::BroadcastLoad>(
```

Comment:
Minor: Extra parentheses after replacing

Reply:
Fixed

Reply:
Actually, they are required. Otherwise, there is an error: 
![image](https://github.com/user-attachments/assets/bef8d9f1-7920-47a3-a804-fc123a886e8f)